benchmark/transpose.cpp

```diff
@@ -247,31 +247,31 @@ static void NvFuserScheduler_Transpose(
       ->Unit(benchmark::kMicrosecond)
 
 NVFUSER_TRANSPOSE_SQUARE_RUN(
-    NF_Transpose_Random_fp32_Inner_2D_01_Axis,
+    NvFuserScheduler_Transpose_Random_fp32_Inner_2D_01_Axis,
```

Comment:
👍 

No other benchmark named like `NF_`?

Reply:
Found a few more when comparing benchmark. I'm still cleaning this up.